Makefile

```diff
@@ -425,6 +425,9 @@ $(CONTROLLER_GEN): $(TOOLS_DIR)/go.mod
 
 controller-gen: $(CONTROLLER_GEN) ## Build controller-gen
 
+protoc: 
+	./scripts/install-protoc.sh
```

Comment:
we don't have a well-defined project layout, but should this scripts dir be under `/build` maybe?

Reply:
From what I have observed most scripts go into $ROOT/scripts (or at least in the kubebuilder project and other go projects). But if build seems like the right place for all scripts then we should definitely move it there. 
@vakalapa chime in?